packages/grafana-ui/src/components/Menu/Menu.tsx

```diff
@@ -44,9 +45,15 @@ export const Menu = React.forwardRef<HTMLDivElement, MenuProps>(
     );
   }
 );
-Menu.displayName = 'Menu';
 
-/** @internal */
+MenuComp.displayName = 'Menu';
+
+export const Menu = Object.assign(MenuComp, {
+  Item: MenuItem,
+  Divider: MenuDivider,
+  Group: MenuGroup,
+});
+
```

Comment:
**Question:** although I actually like this form of grouping sub-components of a certain type, now there are going to be two ways to access `MenuGroup` and `MenuItem` - either importing them directly from `@grafana/ui` or by importing them implicitly and using `Menu.Item`. Do we have any preference? Is this a pattern that we would like to start doing for other components as well?

Reply:
@leventebalogh we are doing this for Card , https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Card/Card.tsx#L28 

yes, introducing this now for MenuItem and MenuGroup does mean they are exported in two ways which is not ideal